deploy/osd-pcap-collector/config.yaml

```diff
@@ -0,0 +1,3 @@
+deploymentMode: "SelectorSyncSet"
+selectorSyncSet:
+  applyBehavior: "CreateOnly"
```

Comment:
We want it to continuously update, so remove this line and use the default apply behavior. 
```suggestion
```